client/js/components/faq/DpFaqSupport.vue

```diff
@@ -0,0 +1,118 @@
+<license>
+(c) 2010-present DEMOS plan GmbH.
+
+This file is part of the package demosplan,
+for more information see the license file.
+
+All rights reserved
+</license>
+
+<template>
+  <div
+    id="scFaq"
+    class="c-support-wrapper space-inset-m">
+    <h2 class="font-normal">
+      {{ Translator.trans('support.heading') }}
+    </h2>
+    <p class="mb-5">
+      {{ Translator.trans('support.introduction') }}
+    </p>
+    <p>
+      {{ Translator.trans('support.contact.advice') }}
+    </p>
+    <h3 class="mt-5">
```

Comment:
```suggestion
    <h3 class="mt-8">
```

Reply:
@elmasdemos @sakutademos these tailwind spacing classes are not yet configured with the demosplan-ui spacing classes. Using them now will most probably lead to visual bugs later when they are configured. 

for now, i'd recommend to use demosplan spacing classes like `u-m-...`.